.eslintrc.json

```diff
@@ -16,7 +17,9 @@
   "rules": {
     "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
     "import/no-unresolved": "off",
+    "jsx-a11y/href-no-hash": "off",
```

Comment:
Opps, we have some little changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json)
You need to fix linter's error.
![image](https://user-images.githubusercontent.com/31897434/73417568-ca44bd00-4318-11ea-8a5e-fedabd629b8c.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

Reply:
Thank you for your feedback @bellom I will work on your comments.

Reply:
1. The live link in the README file seems not working perfectly, could that be from my end thinking ;
DONE

2. I will suggest you disable both the edit and comment buttons because what they do basically is to refresh the page, which is not professional right? (NOTE: edit and comment features are not mandatory but if you can add it, that would be great.)
DONE

3. Kindly working on responsiveness, that's is; mobile view; the input forms are all been joined together, let's have some little margin.
DONE

About the eslint error I carry this error since milestone 3 and previous TSE help me but we could't fix it, perhaps you can help me with the errors